sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -466,13 +467,22 @@ func (m *networkFlowManager) enrichConnection(conn *connection, status *connStat
 		}
 
 		if extSrc == nil {
-			// Fake a lookup result.
+			// Fake a lookup result. This shows "External Entities" in the network graph
 			lookupResults = []clusterentities.LookupResult{
 				{
 					Entity:         networkgraph.InternetEntity(),
 					ContainerPorts: []uint16{port},
 				},
 			}
+			if conn.incoming {
+				log.Debugf("Incoming connection to container %s/%s from %q. "+
+					"Marking it as 'External Entities' in the network graph.",
+					container.Namespace, container.ContainerName, conn.remote.IPAndPort.String()+":"+strconv.Itoa(int(port)))
```

Comment:
```suggestion
				log.Debugf("Incoming connection to container %s/%s from %q:%q. "+
					"Marking it as 'External Entities' in the network graph.",
					container.Namespace, container.ContainerName, conn.remote.IPAndPort.String(), strconv.Itoa(int(port)))
```

Would save a few cycles and memory allocation when the log is disabled.

Reply:
Good point, but I must change `%q` to `%s`, otherwise the log would be `... from "10.0.0.1":"443".` as `%q` is quoting the element.

Reply:
Got that covered in [b66d6cf](https://github.com/stackrox/stackrox/pull/8580/commits/b66d6cf868612ecc2431b945b690cf5c2bdd7ee1)